R/JHU.R

```diff
@@ -0,0 +1,163 @@
+#' R6 Class containing specific attributes and methods for John Hopkins
+#' University data
+#'
+#' @description Country specific information for downloading, cleaning
+#'  and processing covid-19 region data for John Hopkins University.
+# nolint start
+#' @source \url{https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data}
+# nolint end
+#' @export
+#' @concept dataset
+#' @examples
+#' \dontrun{
+#' national <- JHU$new(verbose = TRUE, steps = TRUE, get = TRUE)
```

Comment:
These are both quite complicated classes having some examples (and maybe custom tests based on them) would be helpful both in order to review they work as expected and for users.

Reply:
Asside from the tests automatically provided, what others do you think would be appropriate?

Reply:
check you can filter into the data as you would expect and want?

Reply:
will handle in seperate PR